packages/terra-search-field/CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Unreleased
 
+* Changed
+  * Group Terra Search Field components together and add optional Label
```

Comment:
Can you update the CHANGELOG entry to be similar to this:

https://github.com/cerner/terra-framework/blob/main/packages/terra-worklist-data-grid/CHANGELOG.md

basically a separate entry for every prop added?